Resources/Prototypes/_LostParadise/Entities/Objects/Weapons/Melee/plasma_sword.yml

```
@@ -0,0 +1,213 @@
+- type: entity
+  name: Plasma sword
+  parent: BaseItem
+  id: LPPPlasmaSword
+  description: "A prototype weapon based on plasma technology, a small injection of plasma ignited by an incendiary device forms a blade of fire as long as there is plasma. Special cartridges are required for refueling."
+  components:
+  - type: EmitSoundOnPickup
```

yamllint

[error] wrong indentation: expected at least 3

(indentation)